agent.go

```diff
@@ -471,7 +473,7 @@ func (a *Agent) watchHealthChecks(nodeListCh chan map[string]bool) {
 
 		if checkCount != len(ourChecks) {
 			checkCount = len(ourChecks)
-			a.logger.Printf("[INFO] Now managing %d health checks across %d nodes", checkCount, len(ourNodes))
+			a.logger.Info("Now managing ", "", hclog.Fmt("%d health checks across %d nodes", checkCount, len(ourNodes)))
```

Comment:
Looks like there is an extra `"", ` in `Info()` that isn't adding any information. Would you be mind removing it?